<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>system_station</class>
 <widget class="QWidget" name="system_station">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Adobe Devanagari</family>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane{
	border-width:5px;
	border-color:rgb(112, 161, 159);
	border-style: outset;
	background-color: rgb(132, 171, 208);
	background: transparent;
} 

QTabBar::tab{
	border-color:rgb(48, 104, 151);
	border-bottom-color: #00aa7f;
	border-top-left-radius: 10px;
	border-top-right-radius: 10px;
	max-width:30ex;
    min-width:30ex; 
	min-height:15px;
	font:18px Times New Roman;
	padding: 0px;
}
QTabBar::tab:hover{
	background:rgb(198, 197, 146);
    color:black;
}
QTabBar::tab:selected{
	border-color: black;
	background:green;
	color:white;
}
QTabWidget::tab-bar {
	border-top: 2px solid #E5E5E5;
	border-bottom: 2px solid #E5E5E5;
	border-left:1px solid #E5E5E5;
	alignment: left;
	font-size: 14px;
	background-color:#FFFFFF;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_8" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 212, 169);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_14" stretch="1,1,10,1">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_41">
              <item>
               <spacer name="horizontalSpacer_23">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_22">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>45</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:#ffffff</string>
                </property>
                <property name="text">
                 <string>   站点二维码绑定界面   </string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_24">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_42">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_39" stretch="3,1">
                <property name="spacing">
                 <number>100</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_43">
                  <property name="spacing">
                   <number>5</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_67">
                    <property name="font">
                     <font>
                      <pointsize>16</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: white;  
</string>
                    </property>
                    <property name="text">
                     <string>当前站点:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="StationName">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>180</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(255, 255, 255);</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="placeholderText">
                     <string>StationName</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="query_station">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: white;  
border: 2px groove gray;
border-radius:15px;border:none;
background-color: rgb(0, 166, 172);</string>
                  </property>
                  <property name="text">
                   <string>查询</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_10">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_45">
                <property name="spacing">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QPushButton" name="add_station">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>45</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: white;  
border: 2px groove gray;
border-radius:15px;border:none;
background-color: rgb(0, 166, 172);</string>
                  </property>
                  <property name="text">
                   <string>添加</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="delete_station">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>45</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: white;  
border: 2px groove gray;
border-radius:15px;border:none;
background-color: rgb(0, 166, 172);</string>
                  </property>
                  <property name="text">
                   <string>删除</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget_station">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color:#ffffff</string>
              </property>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>新建行</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>序号</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>站点名称</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>站点二维码</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>起始二维码</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>途经二维码0</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>途经二维码1</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>途经二维码2</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>途经二维码3</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>对接二维码</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>对接路线名称</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>方向</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_46" stretch="10,1">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_25">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_47" stretch="0">
                <property name="spacing">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QPushButton" name="update_station">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>160</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>45</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>16</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: white;  
border: 2px groove gray;
border-radius:15px;border:none;
background-color: rgb(0, 166, 172);</string>
                  </property>
                  <property name="text">
                   <string>提交更新</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
